src/vorta/borg/borg_thread.py

```diff
@@ -142,7 +144,8 @@ def run(self):
         log_entry.save()
         logger.info('Running command %s', ' '.join(self.cmd))
 
-        p = Popen(self.cmd, stdout=PIPE, stderr=PIPE, bufsize=1, universal_newlines=True,
+        extra_args = shlex.split(self.params.get('extra_borg_arguments', ''))
+        p = Popen(self.cmd + extra_args, stdout=PIPE, stderr=PIPE, bufsize=1, universal_newlines=True,
```

Comment:
This appends extra args at the very end. Like `borg create --debug $REPO::$SNAPSHO --rsh=xxx`.

Another option would be to append them in `vorta.borg.borg_thread.prepare_bin()`. Because extra args apply to *all* different `borg` commands, right?

Reply:
They do.

Sure, it'd work either way. I added them at the end to not change the current interface of `prepare_bin()`.

Reply:
please check "order matters" in the borg docs.

Reply:
Why not insert the extra args around [Line 44 in this file](https://github.com/borgbase/vorta/blob/master/src/vorta/borg/borg_thread.py#L44). This is where global options are added. Python also has a helper function to insert elements into a list, like `self.cmd`

```
# (existing) change path to Borg binary, if needed
cmd[0] = self.prepare_bin()

# insert extra args
cmd.insert(1, params.['extra_borg_arguments'])  # TODO: skip if no extra args, split args
```

Reply:
I agree. @ThomasWaldmann you're right about the order of the arguments, I fixed it.

@m3nu I moved constructing the command line to where you suggested. 👍